osu.Game/Tests/Visual/ScreenTestCase.cs

```diff
@@ -33,5 +33,11 @@ protected void LoadScreen(OsuScreen screen)
                 stack.Exit();
             stack.Push(screen);
         }
+
+        protected override void Dispose(bool isDisposing)
+        {
+            stack.Dispose();
```

Comment:
WhaT?

Reply:
what what?

Reply:
We have never before disposed another drawable from a drawable, this shouldn't even do anything different in theory because it's added to the hierarchy and children are already disposed in `Dispose()`.